src/main/java/uk/gov/di/ipv/dto/CredentialIssuers.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.di.ipv.dto;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+public class CredentialIssuers {
+
+    private final Set<CredentialIssuerConfig> credentialIssuerConfigs;
+
+    public CredentialIssuers() {
+        credentialIssuerConfigs = Collections.emptySet();
+    }
+
+    public CredentialIssuers(Set<CredentialIssuerConfig> credentialIssuerConfigs) {
+        this.credentialIssuerConfigs = credentialIssuerConfigs;
+    }
+
+    public Set<CredentialIssuerConfig> getCredentialIssuerConfigs() {
+        return credentialIssuerConfigs;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        CredentialIssuers that = (CredentialIssuers) o;
+        return Objects.equals(credentialIssuerConfigs, that.credentialIssuerConfigs);
+    }
```

Comment:
are we ever comparing `CredentialIssuers`? If not we shouldn't bother with an equals and hashcode method

Reply:
Only in the test shouldLoadCredentialIssuersFromBase64EncodedString() - Happy for this to stay in, or want me to refactor the test?

Reply:
ok 👍 , pls keep it in then 😄